src/api/staking.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+  getDelegatedStakingActivities,
+  getNumberOfDelegators,
+  getNumberOfDelegatorsForAllPools,
+} from "../internal/staking";
+import { GetDelegatedStakingActivitiesResponse, GetNumberOfDelegatorsForAllPoolsResponse, HexInput } from "../types";
+import { AptosConfig } from "./aptos_config";
+
+/**
+ * A class to query all `Staking` related queries on Aptos.
+ */
+export class Staking {
+  readonly config: AptosConfig;
+
+  constructor(config: AptosConfig) {
+    this.config = config;
+  }
+
+  /**
+   * Queries current number of delegators in a pool.  Throws an error if the pool is not found.
+   *
+   * @param poolAddress Pool address
+   * @returns The number of delegators for the given pool
+   */
+  async getNumberOfDelegators(args: { poolAddress: HexInput }): Promise<number> {
```

Comment:
looks like in other count like queries we return `{count:number}`, returning just the number might make more sense.